io-engine/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -344,6 +348,31 @@ impl Display for NexusState {
     }
 }
 
+#[derive(Debug)]
+pub struct ZoneInfo {
+    zoned: bool,
+    num_zones: u64,
+    zone_size: u64,
+    max_zone_append_size: u32,
+    max_open_zones: u32,
+    max_active_zones: u32,
+    optimal_open_zones: u32,
```

Comment:
Would be good to have some doc comments here describing what these mean for the folks who are not very familiar with zones, such as myself :)